src/model/schema.js

```diff
@@ -1165,13 +1163,25 @@ export class SchemaContext {
 
 	/**
 	 * Returns new SchemaContext instance with additional items created from provided definition.
+	 * Definition can be:
+	 *
+	 * 		const ctx = new SchemaContext( element ); // [ 'elementParent', 'element' ]
```

Comment:
We should not use `ctx` in public API, including docs samples. Use `context` instead.